### PR TITLE
Start run_migration command

### PR DESCRIPTION
**Pull request by JeanMarie-TTS**
## run_migration
Implement a django management command to run migrations.
It takes submissions from the raw Census tables in PG and attempts to load them one at a time into the FAC application using existing code from wirkbooklib.
This is intended to be a starting point for us to do a continuous cycle of maing changes including refactoring as we work through all the requirements laid out in the ADR.
Running this command on the existing codebase will throw errors, but it allows us to see what they are and work on them.
`manage.py run_migration --year 2022`

## PR checklist: submitters

- [ ]   Link to an issue if possible. If there’s no issue, describe what your branch does. Even if there is an issue, a brief description in the PR is still useful.
- [ ]   List any special steps reviewers have to follow to test the PR. For example, adding a local environment variable, creating a local test file, etc.
- [ ]   For extra credit, submit a screen recording like [this one](https://github.com/GSA-TTS/FAC/pull/1821).
- [ ]   Make sure you’ve merged `main` into your branch shortly before creating the PR. (You should also be merging `main` into your branch regularly during development.)
- [ ]   Make sure you’ve accounted for any migrations. When you’re about to create the PR, bring up the application locally and then run `git status | grep migrations`. If there are any results, you probably need to add them to the branch for the PR. Your PR should have only **one** new migration file for each of the component apps, except in rare circumstances; you may need to delete some and re-run `python manage.py makemigrations` to reduce the number to one. (Also, unless in exceptional circumstances, your PR should not delete any migration files.)
- [ ]   Make sure that whatever feature you’re adding has tests that cover the feature. This includes test coverage to make sure that the previous workflow still works, if applicable.
- [ ]   Make sure the full-submission.cy.js [Cypress test](https://github.com/GSA-TTS/FAC/blob/main/docs/testing.md#end-to-end-testing) passes, if applicable.
- [ ]   Do manual testing locally. Our tests are not good enough yet to allow us to skip this step. If that’s not applicable for some reason, check this box.
- [ ]   Verify that no Git surgery was necessary, or, if it was necessary at any point, repeat the testing after it’s finished.
- [ ]   Once a PR is merged, keep an eye on it until it’s deployed to dev, and do enough testing on dev to verify that it deployed successfully, the feature works as expected, and the happy path for the broad feature area (such as submission) still works.

## PR checklist: reviewers

- [ ]   Pull the branch to your local environment and run `make docker-clean; make docker-first-run && docker compose up`; then run `docker compose exec web /bin/bash -c "python manage.py test"`
- [ ]   Manually test out the changes locally, or check this box to verify that it wasn’t applicable in this case.
- [ ]   Check that the PR has appropriate tests. Look out for changes in HTML/JS/JSON Schema logic that may need to be captured in Python tests even though the logic isn’t in Python.
- [ ]   Verify that no Git surgery is necessary at any point (such as during a merge party), or, if it was, repeat the testing after it’s finished.

The larger the PR, the stricter we should be about these points.

### CODE REVIEW

File: backend/census_historical_migration/loader.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from audit.models import SingleAuditChecklist
+from .models import ELECAUDITHEADER as Gen
+from .make_submission import load_historic_data
+
+
+def load_data(audit_yaer):
+    SingleAuditChecklist.objects.all().delete()
```
Comment (jadudm) [inline backend/census_historical_migration/loader.py:7]: I know this is still in draft, but this absolutely cannot go to production as-is. This line is terrifying.

**Review by jadudm — CHANGES_REQUESTED** [review]
See my comment on line 7.

I'm happy for us to do some pairing on this code. I would be happy to collaborate on improving it. But we cannot afford to have code going in that has something that might delete all of our data.

Perhaps I'm misreading this line, but that really is what it looks like?

File: backend/census_historical_migration/loader.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from audit.models import SingleAuditChecklist
+from .models import ELECAUDITHEADER as Gen
+from .make_submission import load_historic_data
+
+
+def load_data(audit_yaer):
```
Comment (eazapata23) [inline backend/census_historical_migration/loader.py:6]: `audit_yaer` should be `audit_year`

**Comment by JeanMarie-TTS** [comment]
> See my comment on line 7.
> 
> I'm happy for us to do some pairing on this code. I would be happy to collaborate on improving it. But we cannot afford to have code going in that has something that might delete all of our data.
> 
> Perhaps I'm misreading this line, but that really is what it looks like?

@jadudm The delete line was temporary and is now remove.

**Comment by jadudm** [comment]
I believe all of this work has been pulled into other PRs. Closing. It can always be reopened.

**Pull request closed without merging**